src/events/instances/MemberRoleChanged.ts

```diff
@@ -0,0 +1,64 @@
+import { Client, MessageEmbed } from 'discord.js';
+import { Event, propagateMessage, NotificationType, getGroupRoleEmoji } from '../../utils';
+import { GroupRole, GroupRoleProps, Player } from '@wise-old-man/utils';
+import config from '../../config';
+
+interface MemberActivity {
+  groupId: number;
+  members: {
+    role: GroupRole;
+    previousRole: GroupRole;
+    player: Player;
+  }[];
+}
+
+class MemberRoleChanged implements Event {
```

Comment:
This should technically be "MembersRolesChanged" (plural) (change on router.ts too)